### DISCUSSION TITLE
When will a Protocol 18-compatible sdk be out?

### DISCUSSION THREAD

**Discussion by liannelee** [Synesso/scala-stellar-sdk#499] [General]
Hi there,

Great working SDK so far!

Just curious if anyone is working on a release that is compatible with Protocol 18?

**Comment by Synesso** [comment]
Hi. I won't personally be able to make a release for protocol 18. Because protocol updates are not backwards compatible, this means the SDK is effectively broken unless someone patches the XDR dependency (to v18.0.3) and figures out the breaking changes that the SDK needs to make its own api.

You may be better off adapting to the java sdk sooner rather than later.
